lib/translator/read.cc

```diff
@@ -0,0 +1,183 @@
+#include "read.h"
+
+#include "absl/types/span.h"
+
+#include "common.h"
```

Comment:
nit: read.h and common.h includes can be put in two consecutive lines

Reply:
I think the .h file that you're implementing is supposed to be in its own line at the top https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
oh true my bad. Missed that its read.cc